sdk/ios/Configsum/Configsum/Configsum/Configuration/Environment.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Environment.swift
+//  Configsum
+//
+//  Created by Alexandru Savu on 2017-10-04.
+
+import Foundation
+
+public struct Environment {
```

Comment:
How about an Environment enum with cases for production, test and debug? Then computed properties based on the case.